src/main/java/software/amazon/sns/AmazonSNSExtendedClient.java

```diff
@@ -79,28 +82,30 @@ public AmazonSNSExtendedClient(AmazonSNS snsClient, SNSExtendedClientConfigurati
      * Documentation</a>
```

Comment:
Please update all the Java docs in this file and AmazonSNSExtendedClientBase.java to match java docs in SDK V2.